src/index.js

```diff
@@ -35,6 +35,15 @@ client.getCommand = function(name) {
 
 require("./commands")(client);
 
+function suggestCommands(cmd, cmdList) {
+  var suggestion = didYouMean(cmd, cmdList);
+  if (suggestion) {
+    logger.error();
+    logger.error("Did you mean", clc.bold(suggestion) + "?");
```

Comment:
Don't mix both arguments and string concatenation. Probably easiest to stick with concatenation:

```suggestion
    logger.error("Did you mean "  + clc.bold(suggestion) + "?");
```